egs/swbd/asr1/run.sh

```diff
@@ -174,6 +190,44 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
     done
 fi
 
+# You can skip this and remove --rnnlm option in the recognition (stage 5)
+if [ -z ${lmtag} ]; then
+    lmtag=$(basename ${lm_config%.*})
+fi
+lmexpname=train_rnnlm_${backend}_${lmtag}_${bpemode}${nbpe}
+lmexpdir=exp/${lmexpname}
+mkdir -p ${lmexpdir}
+
+if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
+    echo "stage 3: LM Preparation"
+    lmdatadir=data/local/lm_train_${bpemode}${nbpe}
+    mkdir -p data/local/lm_train ${lmdatadir}
+    cut -f 2- -d" " data/${train_set}/text | gzip -c > data/local/lm_train/${train_set}_text.gz
+    if [ -n "${fisher_dir}" ]; then
+        cut -f 2- -d" " data/train_fisher/text | gzip -c > data/local/lm_train/train_fisher_text.gz
+        zcat data/local/lm_train/${train_set}_text.gz data/local/lm_train/train_fisher_text.gz |\
```

Comment:
did you also add the swbd text data?
You seem to only use fisher text data.

Reply:
swbd and fisher texts are concatenated and encoded at l208, 209.

Reply:
oh, I see.
Then, can you add a comment about it?